flink-core/src/test/java/org/apache/flink/util/SerializedValueTest.java

```diff
@@ -47,26 +50,47 @@ public void testSimpleValue() {
             assertNotNull(v.toString());
             assertNotNull(copy.toString());
 
+            assertNotEquals(0, v.getByteArray().length);
+            assertArrayEquals(v.getByteArray(), copy.getByteArray());
+
+            byte[] bytes = v.getByteArray();
+            SerializedValue<String> saved =
+                    SerializedValue.fromBytes(Arrays.copyOf(bytes, bytes.length));
+            assertEquals(v, saved);
+            assertArrayEquals(v.getByteArray(), saved.getByteArray());
         } catch (Exception e) {
             e.printStackTrace();
             fail(e.getMessage());
         }
     }
 
     @Test
```

Comment:
Might be a bit shorter:

```suggestion
    @Test(expected = NullPointerException.class)
```